Drift/Private/DriftSchemas.h

```diff
@@ -214,6 +217,18 @@ struct FChangePlayerNamePayload
 	bool Serialize(SerializationContext& context);
 };
 
+/*
+ * Serialization of this from JSON to the struct, is handled by FJSonObjectConverter instead of JsonArchive
+ */
+USTRUCT(BlueprintType)
```

Comment:
I don't think this needs to be a `BlueprintType`. Just `USTRUCT()` should be fine. Likewise for the `BlueprintReadOnly`. Just `UPROPERTY()` should be fine. 

Style nit: Should this be `ClientConfigs`? The `FJsonObjectConverter ` has support for converting between `PascalCase` and `snake_case`. If you don't want to mess w/ this I understand though.